apps/test/unit/componentLibrary/SegmentedButtonsTest.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import sinon from 'sinon';
+
+import {expect} from '../../util/reconfiguredChai';
+
+import SegmentedButtons from '@cdo/apps/componentLibrary/segmentedButtons';
+
+const valuesMap = {};
+const onSegmentedButtonsChange = (name, value) => (valuesMap[name] = value);
+
+describe('Design System - Segmented Buttons', () => {
+  it('SegmentedButtons - renders with correct button labels', () => {
+    render(
+      <SegmentedButtons
+        selectedButtonValue="label"
+        buttons={[
+          {label: 'Label', value: 'label'},
+          {label: 'Label2', value: 'label-2'},
+        ]}
+        onChange={value => onSegmentedButtonsChange('label', value)}
+      />
+    );
+
+    const segmentedButton1 = screen.getByText('Label');
+    const segmentedButton2 = screen.getByText('Label2');
+
+    expect(segmentedButton1).to.exist;
+    expect(segmentedButton2).to.exist;
+    expect(valuesMap.label || 'label').to.equal('label');
```

Comment:
is the `|| 'label'` needed here? Wouldn't this make the check pass even if the callback didn't fire? (similarly for the other tests)

Reply:
Fixed, thanks